app/frontend/styles/components/_paginated_filter.scss

```diff
@@ -80,6 +80,17 @@
   }
 }
 
+// Fix for fixed position of layout filters
+.js-enabled .moj-filter-layout__filter {
+  @include govuk-media-query($from: tablet) {
+    position: relative;
+  }
+
+  @include govuk-media-query($from: mobile) {
+    position: relative;
+  }
+}
```

Comment:
That was the only way I saw to override the .moj-filter-layout__filter

Other solution was to use:

```
.app-filter {
@include govuk-media-query($from: tablet) {
    position: relative !important;
  }

  @include govuk-media-query($from: mobile) {
    position: relative !important;
  }
}
```

If you know other way, I am happy to change!